Include/cpython/abstract.h

```diff
@@ -162,6 +162,14 @@ PyAPI_FUNC(PyObject *) _PyObject_VectorcallMethod(
     PyObject *name, PyObject *const *args,
     size_t nargsf, PyObject *kwnames);
 
+static inline PyObject *
+_PyObject_CallMethodNoArgs(PyObject *self, PyObject *name)
+{
+    PyObject *args[1] = {self};
```

Comment:
This looks unnecessary. Do you have a reason to use a one-element `args` instead of just `&self`?

Reply:
For calling conventions passing arguments via registers (which is typical on most systems, but not on 32-bit `x86`), this won't make a difference as the compiler needs a temporary anyway for `self` (you can't have a pointer to a register). But I fixed it anyway.

Reply:
Thank you!
It's `inline`, so a calling convention might not come into play.
A sufficiently smart compiler should figure things out, but more straightforward code shouldn't hurt.